product-scenarios/4-gateway/pom.xml

```diff
@@ -31,61 +31,120 @@
+                            <parallel>false</parallel>
+                            <suiteXmlFiles>
+                                <suiteXmlFile>src/test/resources/testng.xml</suiteXmlFile>
+                            </suiteXmlFiles>
+                            <systemProperties>
+                                <property>
+                                    <!-- Common resource directory (eg: Key stores, certificates, etc.)-->
+                                    <name>common.resources.dir</name>
+                                    <value>
+                                        ${basedir}/../../../resources/
+                                    </value>
+                                </property>
+                                <property>
+                                    <!-- testcase specific  resources directory-->
+                                    <name>test.resources.dir</name>
+                                    <value>
+                                        ${basedir}/src/test/resources/
+                                    </value>
+                                </property>
+                                <property>
+                                    <!-- Directory containing carbon applications -->
+                                    <name>test.resources.carbonApplications.dir</name>
+                                    <value>
+                                        ${basedir}/../../../04-SynapseConfigProject/04-synapseConfigCompositeApplication/target/
+                                    </value>
+                                </property>
+                                <property>
+                                    <!-- Related carbon applications (comma separated) to deploy -->
+                                    <name>test.resources.carbonApplications.list</name>
+                                    <value>
+                                        04-synapseConfigCompositeApplication_1.0.0
+                                    </value>
+                                </property>
+                                <property>
+                                    <name>usedefaultlisteners</name>
+                                    <value>false</value>
+                                </property>
+                            </systemProperties>
+                            <workingDirectory>${basedir}/target</workingDirectory>
+                        </configuration>
+                    </plugin>
+                </plugins>
+            </build>
+        </profile>
+        <profile>
+            <id>profile_artifacts</id>
+            <modules>
+                <module>04-SynapseConfigProject/04-synapseConfig</module>
+                <module>04-SynapseConfigProject/04-synapseConfigRegistry</module>
+                <module>04-SynapseConfigProject/04-synapseConfigCompositeApplication</module>
+            </modules>
+            <build>
+                <plugins>
+                    <plugin>
```

Comment:
If the plugin configurations are the same in both profiles, shall we bring them out of the profiles?

Reply:
Fixed